www/js/config.js

```diff
@@ -17,3 +17,23 @@ export const config = {
   fallingTime: 3000, // ms that a piece spends falling before it's placed
 
 };
+
+// export const config = {
+//   isRealtime: true,
+//   turnTime: 0, // ms
+//   boardSize: 30,
+//   colors: [
+//     'black', 'white', 'steelblue', 'lightgreen', 'pink',
+//     'orange', 'red', 'purple', 'blue', 'gray',
+//   ],
+
+//   startingMana: 25,
+//   maxMana: 50,
+//   manaRegenRate: 10,
+
+//   aiNeighborWeight: 500,
+//   apm: 1500,
+
+//   fallingTime: 30, // ms that a piece spends falling before it's placed
+
+// };
```

Comment:
I assume this was for testing purposes -- can I delete?

Alternatively, could set up this config like the botConfigs and have different named configs for different rulesets